recipes-bsp/imx-uuc/imx-uuc_git.bb

```diff
@@ -12,7 +12,7 @@ PR = "r1"
 PV = "0.5.1+git${SRCPV}"
 
 SRC_URI = "git://github.com/NXPmicro/imx-uuc.git;protocol=https"
-SRCREV = "ebb7d34853133d92eff9766e370ea21e6d6d8f2b"
+SRCREV = "d6afb27e55d73d7ad08cd2dd51c784d8ec9694dc"
```

Comment:
This commit should be sent as another PR as it can go in right now.

Reply:
I've dropped it, would send a patch in a separate PR against `master`